examples/wallet/app/utils/wasmWrapper.js

```diff
@@ -147,47 +122,91 @@ export async function buildTransaction(
   } = await wasmBindings;
   const { calculateFee } = feeCalculator(nodeSettings);
   const privateKey: PrivateKey = PrivateKey.from_bech32(secret);
-  const sourcePublicKey: PublicKey = privateKey.to_public();
-  const sourceAccount: Account = Account.from_public_key(sourcePublicKey);
+  const sourceAccount: Account = Account.from_public_key(
+    privateKey.to_public()
+  );
 
-  // 1 input + 1 output
-  const computedFee = calculateFee(1, 1, 0);
-  const inputAmount = computedFee + amount;
+  // 1 input + 1 certificate
+  const computedFee: number = calculateFee(
+    1,
+    output ? 1 : 0,
+    certificate ? 1 : 0
+  );
+  const iobuilder: InputOutputBuilder = InputOutputBuilder.empty();
+  let inputAmount;
+  if (typeof output === 'object') {
```

Comment:
can you ask if it is of type `Output` or something like that?

Reply:
no, because flow types are removed when compiling

Reply:
although this will evaluate as true for `null`, I'll try to do something else.